src/utils/date.ts

```diff
@@ -1,3 +1,8 @@
-export const normalizeToDateOnly = async (date: Date): Promise<Date> => {
-  return new Date(date.toISOString().split("T")[0]);
+import { format } from "date-fns";
+
+/**
+ * Parse date and return the day of the week
+ */
+export const getWeekdayFromDate = async (date: Date): Promise<string> => {
```

Comment:
I don't see a value in this helper. 
1) It is just one line of the code
2) you actually don't get weekday from date. You format weekday in desirable template. Are you sure that in every place the same template will be used? I'm not sure. If you will need diff template, than template should be an argument, it will complicate function and it will be more simple just to use format. I don't see a problem to write there something like const formattedWeekday =  format(date, "EEEE") and that's it. 
3) you have it in 1 place

Reply:
Makes sense.

It had been much more complex in previous versions, then I discovered `date-fns` and it became much easier.

Helper is replaced with the direct call to `date-fns`. Thanks!